lib/matplotlib/image.py

```diff
@@ -13,17 +13,15 @@
 import PIL.PngImagePlugin
 
 import matplotlib as mpl
-from matplotlib import _api
+from matplotlib import _api, cbook, cm
+# For clarity, names from _image are given explicitly in this module
+from matplotlib import _image
+# For user convenience, the names from _image are also imported into
+# the image namespace
```

Comment:
One can actually question this comment. At least, assuming that the whole idea of putting things into `_image` was to make these functions and classes "private".

Reply:
It's `_image` because it's a compiled extension, not necessarily to be private.

Reply:
`_image` contains C code (see `image_wrapper.cpp`). It's a standard but arguably confusing convention to use a leading underscore for C parts of of a python module.

Reply:
Thanks for the clarification!

I double-checked that nothing is imported from `_image` through `image` in the code base and unless I failed or there were some missing dependency not triggering the code, it should at least be OK from that perspective.

But probably safest to keep it as is for now.